src/requirements/requirement-frontend-computation.ts

```diff
@@ -286,7 +286,7 @@ export default function computeGroupedRequirementFulfillmentReports(
 
   // college may be undefined if the user has only selected a grad program
   if (college) {
-    groupedRequirementFulfillmentReport.push({
+    groupedRequirementFulfillmentReport.unshift({
```

Comment:
I remember you said that order in the `groupedRequirementFulfillmentReport` mattered. Can you add a comment here to briefly explain why we unshift and not push?